db/migrate/20150625172435_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Migrations should not be used for migrating data. For that purpose - create a custom rake task - read more [here](http://guides.rubyonrails.org/command_line.html#custom-rake-tasks)